modules/appnexusAstBidAdapter.js

```diff
@@ -16,10 +16,11 @@ const NATIVE_MAPPING = {
   body: 'description',
   image: {
     serverName: 'main_image',
-    serverParams: {
-      required: true,
-      sizes: [{}]
-    }
+    serverParams: { required: true, sizes: [{}] }
+  },
+  icon: {
+    serverName: 'icon',
+    serverParams: { sizes: [{}] }
```

Comment:
In testing the `required` param didn't seem to affect the response, but did add it in to be consistent with `image` type